chatkit-core/src/test/kotlin/com/pusher/chatkit/rooms/state/JoinedRoomsStateTestUtil.kt

```diff
@@ -31,6 +32,12 @@ internal fun Assert<JoinedRoomsState>.isEmpty() {
     prop(JoinedRoomsState::unreadCounts).isEmpty()
 }
 
+internal fun Assert<JoinedRoomsState>.contains(
+    room: Pair<String, JoinedRoomInternalType>
+) {
+    prop(JoinedRoomsState::rooms).contains(room)
```

Comment:
Doesn't assertk have it's own contains so we don't need our own util method? The RoomUpdatedTest could just use `contains` directly?